### DISCUSSION TITLE
How can I redirect to a page when there is an update in my apk file using cordova mobile of quasar?

### DISCUSSION THREAD

**Discussion by AuliaAmirullah15** [quasarframework/quasar#11630] [CLI - Capacitor / Cordova modes]
I'd like to force logging users out when there is an update installed on their phones. I used service worker and set the local storage I use to null. But it seems not working to redirect users who are already logged in to the login page. How can I solve the problem? 

```
mounted() {
    const forceSWupdate = () => {
      if ("serviceWorker" in navigator) {
        navigator.serviceWorker
          .getRegistrations()
          .then(function (registrations) {
            for (let registration of registrations) {
              registration.update();
            }
            window.localStorage.set("loggedIn", null);
            window.localStorage.set("token", null);
          });
      }
    };
    forceSWupdate();
}
```

**Comment by pikil** [comment]
I think you are mixing PWA and Mobile version here, which might not work. The better way would be to import app version from package.json (that's where you control your version, right?) and compare it agains saved one in LocalStorage.

When there is a mismatch, log the user out! That should do the trick :)
